docs/gethue/content/en/posts/2022-02-01-hplsql-support.md

```diff
@@ -0,0 +1,110 @@
+---
+title: HPL/SQL Support
+author: Hue Team
+type: post
+date: 2022-02-01T00:00:00+00:00
+url: /blog/2022-02-01-hplsql-support
+sf_thumbnail_type:
+  - none
+sf_thumbnail_link_type:
+  - link_to_post
+sf_detail_type:
+  - none
+sf_page_title:
+  - 1
+sf_page_title_style:
+  - standard
+sf_no_breadcrumbs:
+  - 1
+sf_page_title_bg:
+  - none
+sf_page_title_text_style:
+  - light
+sf_background_image_size:
+  - cover
+sf_social_sharing:
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.11
+  - Development
+  - Query
+
+---
+
+HPL/SQL is an Apache open source procedural extension for SQL for Hive users. It has its own grammar. It is a built-in part of Apache Hive since version 2.0.
```

Comment:
Change from "It is a built-in part of Apache Hive since version 2.0."
to "It is included with Apache Hive from version 2.0."